docs/content/docs/grype/_index.md

```diff
@@ -52,17 +50,13 @@ grype scanner back to the legacy scanner is not supported and will cause data is
     helm upgrade
 ```
 
-After making the relevant change above and redeploying, the system will start up with the grype vulnerability scanner enabled and will
-sync the latest version of grype db. Note that legacy feeds will no longer be synced while grype is configured. All vulnerability data
-and scanning will now come from the grype feed.
+After making the relevant change above and redeploying, the system will start up with the legacy vulnerability scanner enabled and will sync the latest version of legacy database. Note that Grype feeds will no longer be synced while legacy is configured. All vulnerability data and scanning will now come from the legacy feed.
```

Comment:
"will no longer be synced" It would be more accurate to say that they will not be synced, as a customer using (and continuing to use) the legacy scanner will not have previously synced grype feed data.

Reply:
"legacy feed" -> "legacy scanner and feed". The configuration for these two things is controlled by the same bit here, but the functionalities in the system are distinct.

Reply:
Hi Dan. I made the changes. Thank you. Dean